storage/tianmu/handler/ha_tianmu.cpp

```diff
@@ -1369,8 +1373,17 @@ int ha_tianmu::fill_row(uchar *buf) {
     std::memcpy(buffer.get(), table->record[0], table->s->reclength);
   }
 
-  for (uint col_id = 0; col_id < table->s->fields; col_id++)
-    core::Engine::ConvertToField(table->field[col_id], *(table_new_iter_.GetData(col_id)), &blob_buffers_[col_id]);
+  for (uint col_id = 0; col_id < table->s->fields; col_id++) {
+    if (core::Engine::ConvertToField(table->field[col_id], *(table_new_iter_.GetData(col_id)),
```

Comment:
please add some comments for this piece of code.

Reply:
ack，I have fixed it.